jsonnet/prometheus-operator/prometheusagents-crd.json

```diff
@@ -3652,6 +3652,10 @@
                     "description": "When true, Prometheus ignores the timestamps for all the targets created from service and pod monitors. Otherwise the HonorTimestamps field of the service or pod monitor applies.",
                     "type": "boolean"
                   },
+                  "overrideTrackTimestampsStaleness": {
+                    "description": "When true, Prometheus ignores the timestamps for all the targets created from service and pod monitors. Otherwise the TrackTimestampsStaleness field of the service or pod monitor applies.",
```

Comment:
The description here needs and update.

Reply:
Remove `overrideTrackTimestampsStaleness`.
Please re-check.